packages/EasyLogging/src/Logger.php

```diff
@@ -249,7 +250,9 @@ private function getAppLogger(): PsrLoggerInterface
         $handlers = $this->handlers;
 
         // Add Bugsnag handler only if enabled
-        if ($this->app->make('config')->get('bugsnag.enabled', false)) {
+        /** @var ConfigRepository $configRepository */
+        $configRepository = $this->app->make('config');
+        if ($configRepository->get('bugsnag.enabled', false)) {
```

Comment:
Is the `ConfigRepository` type correct here? I assumed it from some "config" related docblocks.